deploy/packages/bbp-packages.yaml

```diff
@@ -62,6 +62,7 @@ packages:
       - touchdetector@5.0.1
       - touchdetector@5.1.0
       - brayns@0.8.0 +opendeck +brion ^ospray@1.7.3
+      - brayns@immersive +opendeck +brion +optix
```

Comment:
In addition to this, we should updated `deploy/configs/applications/modules.yaml` with:

```
      - brayns@0.8.0
      - brayns@immersive`
```